RELEASE_NOTES.md

```diff
@@ -1,5 +1,39 @@
 # Release Notes
 
+# Staging 2019/10/30
+## Secondary Analysis
+### Lira: v0.22.3 -> v0.22.6
+* Pass pipeline version from the Lira config to the workflow inputs
+* Add monitoring image from the Lira config to the workflow options (if defined)
+* Fix monitoring image set-up to record workflow metadata in BigQuery
+* Fix path to final workflow log directory
```

Comment:
Oh wait @rhiananthony this should include all of the changes since v0.22.3 right?

Reply:
I verified that all of the release notes from 22.4, 22.5, 22.6. What seems to be missing?

Reply:
You're right!